db/program.go

```diff
@@ -186,6 +189,22 @@ func (d *DB) CreateProgram(c echo.Context) error {
 		p.Name = requestBody.Prog.Name
 	}
 
+	wid := requestBody.WID
+	var cid string
+	var class *Class
+	if wid != "" {
```

Comment:
Should I add some sort of security check here?

Reply:
The editor has never really had a whole lot of "security" 😬 

I'd say this is sufficient for the moment.